main.js

```diff
@@ -0,0 +1,30 @@
+var amountInput = document.querySelector('.amountInput');
+var amountEntered = function(event) {
+  event.target.value
+}
+amountInput.addEventListener('keyup', amountEntered)
+
+var nameInput = document.querySelector('.nameInput');
+var button = document.querySelector('#add-expense');
+var total = document.querySelector('.expense-total');
+
+
+
+var handleButtonClick = function(event){
+  // add Expense Input into new tr/td
+  var newTR = document.createElement('tr');
+
+  var nameTD = document.createElement('td');
+  nameTD.textContent = nameInput.event.target.value;
```

Comment:
Same as line 22. `nameInput` doesn't have an `event` property. You're incorrectly calling that.  The event is passed in and contains a reference to the dom node that triggered it as `event.target`